Source/ProgressionSystemRuntime/Private/Components/PSSpotComponent.cpp

```diff
@@ -47,6 +47,50 @@ void UPSSpotComponent::OnReset_Implementation()
 	}
 }
 
+// Listen game states to switch character skin. 
+void UPSSpotComponent::OnGameStateChanged_Implementation(ECurrentGameState CurrentGameState)
+{
+	// it's possible that spot might not be loaded till that time so no ensure added
+	UPSSpotComponent* CurrentSpot = UPSWorldSubsystem::Get().GetCurrentSpot();
```

Comment:
can be `const`: `const UPSSpotComponent* CurrentSpot ... `

Reply:
done https://github.com/moinm3uw/ProgressionSystem/pull/63/commits/9ba9bb463a73890019a0174d0cd0da39a08d102c